### PR TITLE
Introduce new prop on humanizetime component

### PR DESCRIPTION
**Pull request by TPRobots**
Pull request opened by github-pullrequestcreator.

### CODE REVIEW

File: src/time.tsx [hunk 1]
```diff
@@ -51,11 +51,12 @@ export const TimeAgo = ({ date }: TimeAgoProps) => {
 
 export interface HumanizeTimeProps {
   milliseconds: number;
+  type?: string;
 }
 
-export const HumanizeTime = ({ milliseconds }: HumanizeTimeProps) => {
+export const HumanizeTime = ({ milliseconds, type = 'approximate-time' }: HumanizeTimeProps) => {
```
Comment (gdgunnars) [inline src/time.tsx:57]: The style name has changed
ref: https://github.com/catamphetamine/javascript-time-ago/blob/master/CHANGELOG.md#214--06102020

File: src/__tests__/time.test.tsx [hunk 2]
```diff
@@ -95,3 +95,13 @@ test('Renders an incorrect Danish humanized representation of a time duration',
...
+
+test('Renders a short representation of a time duration using a style', () => {
+  const { getByText } = render(
+    <LocalizationProvider locale="en-US" translations={{}}>
+      <HumanizeTime milliseconds={11 * 1000 * 60} type={'mini-minute'} />
+    </LocalizationProvider>
+  );
+
+  expect(getByText('11m')).toBeDefined();
+});
```
Comment (sallas) [inline src/__tests__/time.test.tsx:107]: missing a new line

**Review by sallas — DISMISSED** [review]
this is a good change

**Comment by madsgodvinjensen** [comment]
I generally approve of the changes. But I would like that new prop to be documented in the readme before merging.

**Comment by gdgunnars** [comment]
@madsgodvinjensen 
I've updated the readme and fixed a tiny issue regarding example use case of that component as well.

Please take a look when you have time.

**Review by madsgodvinjensen — APPROVED** [review]

**Pull request merged**